packages/terra-time-input/src/_AccessibleValue.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  // The value you want read only when changed. An undefined value is never read.
+  value: PropTypes.string,
+  readThis: PropTypes.string,
+};
+
+const defaultProps = {
+  value: undefined,
```

Comment:
Default props of `undefined` are unnecessary. If a prop is not defined it will have an undefined value.

Reply:
I see. I'll change it.